pyrefinebio/__init__.py

```diff
@@ -22,3 +22,9 @@
     NotFound,
     DownloadError
 )
+
+from pyrefinebio.hlf import (
```

Comment:
I don't love having an abbreviation as a module name. I think spelling out `high_level_functions` is more verbose but 
far more clear for new contributors.